ext/semian/tickets.c

```diff
@@ -66,6 +66,7 @@ update_ticket_count(update_ticket_count_t *tc)
     }
   }
 
+  // This will update the ctime
```

Comment:
this is very important to understand, as it's the magic that makes this all work.